2b_portfolio_prep_supervisor_workflow.R

```diff
@@ -10,6 +10,8 @@ prep_pf <- raw_pf %>%
   add_holding_id() %>%
   rename(current_shares_outstanding_all_classes = current_shares_outstanding)
 
+prep_pf <- add_meta_portfolio(prep_pf, inc_meta_portfolio)
```

Comment:
I honestly don't know what `add_meta_portfolio()` is or what it does, nor what `inc_meta_portfolio` is in this context, but assuming it does what you want, this makes total sense to me.

Reply:
It simply duplicates the entire portfolio and adds investor_name = "Meta Investor" and portfolio_name = "Meta Portfolio" for every row in the duplicate, so that the entire input can be processed as one summary analysis across investors and portfolios. This is only relevant in offline PACTA I think, since it is a common case to process many portfolios by many investors at the same time in that scenario.